lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -195,6 +196,24 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
     }
   end
 
+  # Some elevators are closed but there are relevant backups
+  defp elevator_closures(
+         [],
+         _active_closures,
+         upcoming_closures,
+         %ElevatorConfig{elevator_id: _elevator_id} = app_params,
+         now,
+         stop_id
+       ) do
+    %ElevatorClosures{
+      app_params: app_params,
+      now: now,
+      station_id: stop_id,
+      stations_with_closures: :nearby_redundancy,
```

Comment:
nit: Since this code path is almost the same as the other one, I think it would be cleaner to combine them and use a `case` expression to compute just this one item.

Reply:
I opted for a ternery-ish statement